### PR TITLE
Use the correct text in Italian faq25.

### PR DESCRIPTION
**Pull request by bfabio**

### CODE REVIEW

File: locales/it/faq/faq25.md [hunk 1]
```diff
@@ -1 +1,6 @@
-I metodi di riconoscimento basati sulla biometria si sono rivelati nel tempo più sicuri delle password numeriche o alfa-numeriche, spesso vulnerabili o involontariamente comunicate ad altre persone. Il riconoscimento biometrico ti offre il vantaggio di velocizzare l'autenticazione, rispetto all'inserimento del Codice di sblocco, e ti permette di svolgere più velocemente le operazioni all'interno dell'app IO.
\ No newline at end of file
+No, i messaggi ricevuti non possono essere cancellati dall'app.
+
+Per questo abbiamo creato la sezione ‘*Archiviati*’, dove puoi spostare i messaggi
```
Comment (Undermaken) [inline locales/it/faq/faq25.md:3]: Thank you @bfabio 
Could you remove line breaks in the second block?

Comment (bfabio) [reply locales/it/faq/faq25.md:2]: @Undermaken Done :+1:

File: locales/it/faq/faq25.md [hunk 2]
```diff
@@ -1 +1,5 @@
-I metodi di riconoscimento basati sulla biometria si sono rivelati nel tempo più sicuri delle password numeriche o alfa-numeriche, spesso vulnerabili o involontariamente comunicate ad altre persone. Il riconoscimento biometrico ti offre il vantaggio di velocizzare l'autenticazione, rispetto all'inserimento del Codice di sblocco, e ti permette di svolgere più velocemente le operazioni all'interno dell'app IO.
\ No newline at end of file
+No, i messaggi ricevuti non possono essere cancellati dall'app.
+Per questo abbiamo creato la sezione ‘*Archiviati*’, dove puoi spostare i messaggi
```
Comment (Undermaken) [inline locales/it/faq/faq25.md:2]:
```suggestion
Per questo abbiamo creato la sezione ‘*Archiviati*’, dove puoi spostare i messaggi che non vuoi più vedere tra quelli ricevuti. In questo modo, se in futuro avrai bisogno di trovare informazioni contenute in un vecchio messaggio, puoi cercare direttamente nella scheda ‘*Archiviati*’.
```
These lines seem to be still trunked

Comment (bfabio) [reply locales/it/faq/faq25.md]: Oops, you're right. Fixed.

**Pull request merged**